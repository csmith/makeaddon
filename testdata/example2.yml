package-as: SomeAddon
manual-changelog: changelog.txt
wowi-archive-previous: yes


externals:
  Libs/LibStub:
    url: svn://svn.wowace.com/wow/libstub/mainline/trunk
    tag: latest
  Libs/CallbackHandler-1.0:
    url: svn://svn.wowace.com/wow/callbackhandler/mainline/trunk/CallbackHandler-1.0
    tag: latest
  Libs/UTF8: svn://svn.wowace.com/wow/utf8/mainline/trunk
  Libs/LibCustomGlow-1.0:
    url: https://github.com/Stanzilla/LibCustomGlow

  Libs/ChatThrottleLib: svn://svn.wowace.com/wow/chatthrottlelib/mainline/trunk


move-folders:
  SomeAddon/SomeAddonOptions: SomeAddonOptions
