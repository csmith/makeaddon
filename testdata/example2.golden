{
  "PackageAs": "SomeAddon",
  "ManualChangelog": {
    "Filename": "changelog.txt",
    "MarkupType": "plain"
  },
  "Externals": {
    "Libs/CallbackHandler-1.0": {
      "Url": "svn://svn.wowace.com/wow/callbackhandler/mainline/trunk/CallbackHandler-1.0",
      "Tag": "latest"
    },
    "Libs/ChatThrottleLib": {
      "Url": "svn://svn.wowace.com/wow/chatthrottlelib/mainline/trunk",
      "Tag": ""
    },
    "Libs/LibCustomGlow-1.0": {
      "Url": "https://github.com/Stanzilla/LibCustomGlow",
      "Tag": ""
    },
    "Libs/LibStub": {
      "Url": "svn://svn.wowace.com/wow/libstub/mainline/trunk",
      "Tag": "latest"
    },
    "Libs/UTF8": {
      "Url": "svn://svn.wowace.com/wow/utf8/mainline/trunk",
      "Tag": ""
    }
  },
  "MoveFolders": {
    "SomeAddon/SomeAddonOptions": "SomeAddonOptions"
  },
  "Ignore": null,
  "LicenseOutput": "",
  "NoLibCreation": true
}